### ISSUE TITLE
Do we support Datacamp materials? 

### ISSUE THREAD

**Issue by maneesha** [datacarpentry/r-intro-geospatial#161]
A few years ago after an incident with Datacamp, The Carpentries [announced](https://carpentries.org/blog/2019/04/datacamp-response/) it would be removing references to Datacamp from its materials.  

The [Introduction to Visualization](https://datacarpentry.org/r-intro-geospatial/07-plot-ggplot2.html) includes links to https://www.statmethods.net/ that now redirect to Datacamp materials.

**Comment by kristi-sara** [comment]
Good point. I started a pull request on removing the statsmethod.net links
